lib/mdx.ts

```diff
@@ -6,12 +6,14 @@ import remarkStringify from 'remark-stringify';
 
 import compilers from '../processor/compile';
 import readmeToMdx from '../processor/transform/readme-to-mdx';
+import divTransformer from '../processor/transform';
 
 export const mdx = (tree: any, { hast = false } = {}) => {
   const processor = unified()
     .use(hast ? rehypeRemark : undefined)
     .use(remarkMdx)
     .use(remarkGfm)
+    .use(divTransformer)
```

Comment:
i'm not 100% on this placement, whether or not it should be before the `hast`ifying